api/v1beta1/placementapi_webhook.go

```diff
@@ -103,12 +115,21 @@ func (r *PlacementAPI) ValidateCreate() (admission.Warnings, error) {
 // ValidateUpdate implements webhook.Validator so a webhook will be registered for the type
 func (r *PlacementAPI) ValidateUpdate(old runtime.Object) (admission.Warnings, error) {
 	placementapilog.Info("validate update", "name", r.Name)
+	errors := field.ErrorList{}
```

Comment:
the empty list set here is unused and overwritten by the line `errors = r.Spec.ValidateUpdate(oldPlacement.Spec, field.NewPath("spec"))`

Reply:
oh right, I don't need this. Removing